docs/custom_text_ops.md

```diff
@@ -0,0 +1,365 @@
+## Operator Schemas
+
+### Auxiliary String Operator
+
+|**Operator**|**Support State**|
+|------------|-----------------|
+|StringEqual |  Supported        |
+|StringHash  |  Supported        |
+|StringToHashBucketFast|Supported|
+|StringJoin  | Supported         |
+|StringRegexReplace| Supported  |
+|StringSplit | Supported       |
+|StringUpper  | Supported     |
+|StringSlice | In next version |
```

Comment:
We don't set up the version yet, so 'in next version' could be 'ongoing'?

Reply:
Sure, fixed.